scss/civicrm/dashboard/dashlets/_getting-started.scss

```diff
@@ -0,0 +1,19 @@
+#civicrm-getting-started {
+  #help {
+    margin: 0;
+  }
+
+  table {
+    border: 0;
+    margin: 7px 0;
```

Comment:
Cant we use existing variables? The margin/padding should be uniform in Shoreditch.

Reply:
will use the value from above.